### PR TITLE
Verify bbss.hackclub.com

### PR DESCRIPTION
**Pull request by saumilthecode**
# [Adding/Deleting/Updating] `subdomain.[hackclub.com/dino.icu/etc]`
added bbss.hackclub.com
## Description
route https://bbsshackclub.vercel.app/ to https://bbss.hackclub.com (since anything ending with hackclub.com will give us more legitimacy) 
Provide a description here of the changes, and if adding a subdomain provide a rationale & what it will be used for.
added subdomain bbss.hackclub.com so that we can use it for our hack club.

### CODE REVIEW

**Comment by saumilthecode** [comment]
i think i have to do this alphabetically

File: hackclub.com.yaml [hunk 1]
```diff
@@ -552,6 +552,13 @@ bank-wrapped:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+bbss:
+  _vercel:
+  ttl: 600
+  type: TXT
+  values:
+    - vc-domain-verify=bbss.hackclub.com,6c79bb5ffee8bbfd78d2
+
```
Comment (Muirrum) [inline hackclub.com.yaml:561]: Please add this verification string to the _vercel record at the top of the file.

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (saumilthecode) [reply hackclub.com.yaml:561]: Okie

**Review by Muirrum — APPROVED** [review]
lgtm

**Comment by saumilthecode** [comment]
heyo! i need to make another change!
do i reopen this or make another pr
image

**Pull request merged**